lib/info.js

```diff
@@ -75,6 +75,9 @@ module.exports = function getInfo(link, options, callback) {
       // Get description.
       description: util.getVideoDescription(body),
 
+      // Get extra info.
+      extras: util.getVideoExtras(body),
```

Comment:
Hmm, not completely sure about the name `extras`. How about `media`, it's always a song, game, or movie right?

Reply:
Tbh the only reason I called it `extras` is because of the YouTube Tag itself being called `<div id="watch-description-extras">`. But sure, I'll rename it to media. Makes more sense tbh, although I've seen a "License" field in some videos but I have yet to test it on one of those but I guess that could be considered media aswell lol.